hw/top_earlgrey/data/ip/chip_alert_handler_testplan.hjson

```diff
@@ -0,0 +1,268 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_alert_handler
+  testpoints: [
+    // ALERT_HANDLER (pre-verified IP) integration tests:
+    {
+      name: chip_sw_alert_handler_alerts
+      desc: '''Verify all alerts coming into the alert_handler.
+
+            An automated SW test, which does the following (applies to all alerts in all IPs):
+            - Program the alert_test CSR in each block to trigger each alert one by one.
+            - Ensure that all alerts are properly connected to the alert handler and cause the
+              escalation paths to trigger.
+
+            Note for silicon validation:
+            - Ensure this test makes no alterations to OTP (use NMI escalation path).
+            '''
+      features: ["ALERT_HANDLER.ALERT.OBSERVE",
+                 "ALERT_HANDLER.ALERT.ESCALATE"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_alert_test"]
+    }
+    {
+      name: chip_sw_alert_handler_escalations
+      desc: '''Verify all alert escalation paths.
+
+            Verify all escalation paths triggered by an alert.
+            - Verify the first escalation results in NMI interrupt serviced by the CPU.
+            - Verify the second results in device being put in scrap state, via the LC JTAG TAP.
+            - Verify the third results in chip reset.
+            - Ensure that all escalation handshakes complete without errors.
+              - This cannot be checked in sival
```

Comment:
Could you be more specific about what cannot be checked in sival? A reset can be, and probably the NMI also. The secret wipes probably cannot.

Reply:
I read this as using probes into the design to observe the handshake signals working correctly. The results of the handshakes are separately covered in the above bullet points. I'll add some extra wording to make this clear.